frontend/src/tests/pages/UCSBDates/UCSBDatesIndexPage.test.js

```diff
@@ -0,0 +1,87 @@
+import { render, waitFor } from "@testing-library/react";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+import UCSBDatesIndexPage from "main/pages/UCSBDates/UCSBDatesIndexPage";
+
+
+import { apiCurrentUserFixtures } from "fixtures/currentUserFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import { ucsbDatesFixtures } from "fixtures/ucsbDatesFixtures";
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+import mockConsole from "jest-mock-console";
+
+describe("UCSBDatesIndexPage tests", () => {
+
+    var axiosMock = new AxiosMockAdapter(axios);
```

Comment:
axiosMock is constant in file, frontend/src/tests/pages/AdminUsersPage.test.js. Maybe we should make it constant here as well

Reply:
good catch. I did this as a global change in the code base